Coda 323 (#330)
canonical/open-documentation-academy: website/our-vision.md

# Our ambition for the Open Documentation Academy

We create opportunities for people who are new to open source by connecting them with documentation tasks on established projects. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment.

With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.

## Our vision

> __Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__

Reviewer: Accessibility has different connotations as well and we need not confuse the reader by focusing on that. I'd recommend the following for communicating CODA's vision/mission/ambition :

__Make open source beginner-friendly__

Our objective is to provide a model and a space for open source and documentation.

Followed by the existing content:

We focus on documentation contributions because they present an opportunity for the public to share their direct experience with software to its maintainers, it has the lowest barrier to entry, it is low risk, and because documentation is always evolving and growing in projects. By contributing to documentation, members get closer to the software itself, which is good preparation if they want to engage as developers, too.
...

# Our ambition for the Open Documentation Academy

We create opportunities for people who are new to open source by connecting them with documentation tasks on established projects. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment.

With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.

> ---

Reviewer 2: Let's remove all line separators to make it look more like a single document.

Author: Done.

## Our vision

> __Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__
> 

Reviewer 2: We can remove this part.
```suggestion
```

Author: Hi @activus-d, I think a highlighted main point would help readers grasp the essence of the para to come. Can we leave this in for 1 more reviewer to look at?

Reviewer 2: Yes, we can! Another community member will review the PR soon. Great job so far @NaraHita

> At CODA—the Canonical Open Documentation Academy—we envision a future where *quality documentation forms the foundation and lifeblood of open-source ecosystems*. We see a world where *clear, accessible, and thoughtful content demystifies technology*. This environment welcomes both newcomers and experienced contributors from every corner of the globe. Together, we commit to build a collaborative and innovative community where open source thrives for generations to come.

Reviewer 2: Let’s introduce the full name first and put the abbreviation in parentheses. After that, we can use the abbreviation `CODA` throughout the rest of the document.

Also, we can make it more concise by removing the last sentence.

```suggestion
At the Canonical Open Documentation Academy (CODA), we envision a future where quality documentation is the foundation of open-source ecosystems. We see a world where clear, accessible, and thoughtful content demystifies technology and creates an environment that welcomes contributors of all experience levels.
```

Author: The last sentence feels important to me. Can we leave it for one more reviewers to look at. Thnx.

Reviewer 2: That’s fine, @NaraHita. Another community member will review the PR soon. Great job so far!

> ---

Reviewer 2: Let's remove all line separators.

Author: Done.

## Our mission

To build the premier global community advancing open-source documentation excellence through mentorship and collaboration.

> As a pioneering program focused on the craft of documentation, the Open Documentation Academy:

Reviewer 2: We can do without the word "premier" here:

> To build a global community advancing open-source documentation excellence through mentorship and collaboration.
> 
> As a pioneering program focused on the craft of documentation, the Open Documentation Academy:

Author: Done.

- **Enables** members to build skills in a welcoming and nuturing environment  
- **Cultivates** a thriving worldwide community through workshops, regular sessions, and personalised guidance  
- **Delivers** clear learning paths, systematic support, and visible recognition to help participants become proficient documentation creators  
- **Removes obstacles** to open-source engagement by establishing documentation as the primary entry pathway

> ---

Reviewer 2: Let's remove all line separators.

Author: Done.

# Open Documentation Academy

__Document. Learn. Contribute. Lead: Discover open source through better documentation.__

## Our Vision

__Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__

At CODA, Canonical Open Documentation Academy, we envision a future where quality documentation is the foundation and lifeblood of open-source ecosystems—a world where clear, accessible, and thoughtful content demystifies technology, inviting both newcomers and experienced contributors from every corner of the globe to collaborate, innovate, and ensure open-source thrives for generations to come.

> ---

Reviewer 2: Can we stick to @degville's version for the heading and introduction?

> # Our ambition for the Open Documentation Academy
> 
> We create opportunities for people who are new to open source by connecting them with documentation tasks on established projects. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment.
> 
> With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.

> ## Our Mission

Reviewer 2: We use sentence case for headings. Please see the [Headings](https://docs.ubuntu.com/styleguide/en/#headings) section of the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en/)

---

## Our Core Values

Anyone who participates in a CODA space or works on a CODA task is a CODA member, and we aim to collaborate with projects that share our values. By participating, you agree to uphold the spirit of the [Ubuntu code of conduct](https://discourse.ubuntu.com/t/ubuntu-code-of-conduct-v2-0/33111/1), champion the CODA values described below, and help set the standard for open-source collaboration.

**Transparency**  
Our work is conducted openly. Documentation, workflows, and community platforms are freely available to anyone seeking to improve documentation.

**Diversity**  
We welcome and celebrate contributors from all backgrounds and geographies. Participation is encouraged regardless of prior expertise or origin.

**Partnership**  
We thrive through collective effort. Through mentoring, peer assistance, and shared knowledge, we strengthen each other's capabilities in a safe and supportive environment.

**Quality**  
We pursue industry-leading documentation standards, supporting contributors at their current level and facilitating professional development through experience in established projects.

> ---

Reviewer 2: Let’s create a new page titled “Our values” for this section, as suggested by @degville in his branch. We can place the new page after the "Our vision" page.

## Our Vision for Your Journey at CODA

> ![Pathway Diagram](media/open-documentation-academy-pathway.png)

Reviewer 2: As mentioned earlier, we use sentence case for headings. Please see the [Headings](https://docs.ubuntu.com/styleguide/en/#headings) section of the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en/).

Also, it would be nice to have a blank line after every heading. This makes it clearer to read the markdown version.
